gallery/src/styles/NxVulnerabilityScore/NxVulnerabilityScoreListExample.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxList, NxVulnerabilityIndicator, formatVulnerabilityScore } from '@sonatype/react-shared-components';
+
+const NxVulnerabilityScoreListExample = () =>
+  <NxList>
+    <NxList.Item>
+      <NxVulnerabilityIndicator severityRating="none" />
+      <NxList.Text className="nx-vulnerability-score">{formatVulnerabilityScore(0)}</NxList.Text>
+      <NxList.Subtext>
+        In this row, the vulnerability score value is the only content in
+        the <code className="nx-code">.nx-list__text</code>, so there's no need to make a separate element;
+        the <code className="nx-code">.nx-vulnerability-score</code> class can be applied directly to
+        the <code className="nx-code">.nx-list__text</code>.
+      </NxList.Subtext>
+    </NxList.Item>
+    <NxList.Item>
+      <NxVulnerabilityIndicator severityRating="low" />
+      <NxList.Text>
+        <span className="nx-vulnerability-score">{formatVulnerabilityScore(1)}</span> is the vulnerability (CVSS) score.
+      </NxList.Text>
+      <NxList.Subtext>
+        In this row, there is other content aside from the vulnerability score value within
+        the <code className="nx-code">.nx-list__text</code>, so a separate element is needed in order
+        to ensure that the <code className="nx-code">.nx-vulnerability-score</code> styles are applied only to the
+        vulnerability score value itself. Note that as of RSC 2.0.0, nx-vulnerability-score does not actually
+        have any styles that are distinct from the rest of the main list text. That may change in the future however, so
```

Comment:
Talking about an ancient version of RSC that predates this component doesn't make a ton of sense.
```suggestion
        vulnerability score value itself. nx-vulnerability-score does not actually
        have any styles that are distinct from the rest of the main list text. That may change in the future however, so
```